hpx/parallel/util/prefetching.hpp

```diff
@@ -253,9 +253,9 @@ namespace hpx { namespace parallel { namespace util
         HPX_FORCEINLINE void prefetch_addresses(T const& ... ts)
         {
             int const sequencer[] = {
-                (_mm_prefetch(
+                0, (_mm_prefetch(
```

Comment:
For the non-x86 code path, you may want to use `__builtin_prefetch`, which both GCC and Clang support.

Reply:
We can do that, MSVC however supports `_mm_prefetch` only.